internetarchive/cli/ia_copy.py

```diff
@@ -36,12 +36,11 @@
                                    will not be saved to history/files/$key.~N~
                                    [default: True].
 """
-from __future__ import print_function, absolute_import
 import sys
+import urllib.parse
```

Comment:
I'd rather use `from urllib.parse import quote` and adjust the usages of `urllib.parse.quote` to `quote`.